requirements-dev.txt

```diff
@@ -1,16 +1,16 @@
 attrs>=19.3.0
-sphinx_compas_theme>=0.9
-sphinx>=1.6
-invoke>=0.14
+autopep8
```

Comment:
do we use this anywhere? should we not just replace with `black`?

Reply:
We haven't officially made the switch to black yet. I didn't want to mix the switch to black with this PR, i can send an immediate follow up with switch plus a general run of black over the entire code.

Reply:
Ah totally at your discretion. I view the move to black as merely changing from `autopep8` to `black` in `reqirements.txt` . Dunno if I'd run `black` on everything at once. Would be a frightening PR to review ;)

Reply:
Ok, then let's do the switch to `black` in this PR, and I will check how scary a full run of black is for a single PR.